packages/catalog-search/src/LearningTypeRadioFacet.jsx

```diff
@@ -28,6 +31,12 @@ const LearningTypeRadioFacet = ({ enablePathways }) => {
     } else {
       dispatch(setRefinementAction('content_type', [type]));
     }
+    if (trackingName) {
```

Comment:
is `trackingName` already available?

Reply:
@muhammad-ammar Yes it is available. We are getting `trackingName` directly from `frontend-app-learner-portal-enterprise` via `SearchContext`

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/f6f56486-b83a-47cb-9d98-54d93aa366c6">